### PR TITLE
Virtual Box Cheat sheet

### PR DESCRIPTION
**Pull request by sanjusam00427**
Virtual Box Cheat sheet , containing keyboard shortcuts of vbox

https://duck.co/ia/view/virtual_box_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/virtualbox.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
+{
+  "id": "virtual_box_cheetsheets",
+  "name": "Oracle Virtual Box",
+  "description": "Keyboard shortcuts for virtualbox, ",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/virtualbox.json]: change to `"description": "Keyboard shortcuts for virtualbox",`

**Comment by gautamkrishnar** [comment]
@sanjusam00427 welcome to ddg. please update your description with the ia page link:https://duck.co/ia/view/virtual_box_cheatsheet

File: share/goodie/cheat_sheets/json/virtualbox.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
+{
+  "id": "virtual_box_cheetsheets",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/virtualbox.json]: change to `"id": "virtual_box_cheetsheets",`

**Comment by sanjusam00427** [comment]
@gautamkrishnar  , https://duck.co/ia/view/virtual_box_cheatsheet , how do i change it ?

**Comment by gautamkrishnar** [comment]
screenshot 200
@sanjusam00427

File: share/goodie/cheat_sheets/json/virtualbox.json (hunk 2 above)
Comment (jophab) [inline share/goodie/cheat_sheets/json/virtualbox.json]: @sanjusam00427 @gautamkrishnar the id should be "virtual_box_cheat_sheet"

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/virtualbox.json]: the id should match the ia page url @jophab

**Comment by moollaza** [comment]
@sanjusam00427 @gautamkrishnar I've corrected the `id` of this IA Page and the links here to `virtual_box_cheat_sheet`

@sanjusam00427 please update the ID in the JSON to match that :+1:

File: share/goodie/cheat_sheets/json/virtualbox.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
+{
+  "id": "virtual_box_cheetsheet",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/virtualbox.json]: Should be `virtual_box_cheat_sheet`

Also, the file-name should be `virtual-box.json`.

File: share/goodie/cheat_sheets/json/virtualbox.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  "name": "Oracle Virtual Box",
+  "description": "Keyboard shortcuts for virtualbox",
+  "aliases": [
+    "virtualbox",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/virtualbox.json]: Please fix the indenting throughout the file. You should be using 4 spaces per indent.

File: share/goodie/cheat_sheets/json/virtualbox.json [hunk 5]
```diff
@@ -0,0 +1,82 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/virtualbox.json]: @sanjusam00427 you need to update the filename to match the id: `virtual-box.json`

File: share/goodie/cheat_sheets/json/virtualbox.json [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+      "virtualbox",
+      "oracle virtualbox",
+      "vbox"
+  ],
+  "template_type": "keyboard",
+  "section_order": [
+      "Common Shortcuts"
+  ],
+  "sections": {
+    "Common Shortcuts": [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/virtualbox.json]: I'd like to see this split into 2+ categories to improve the readability
 Add a line note

File: share/goodie/cheat_sheets/json/virtualbox.json (hunk 5 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/json/virtualbox.json]: That will force the default trigger to be `virtual box`. With the `aliases` you have below, `virtualbox cheat sheet` will also trigger this :+1:

**Comment by moollaza** [comment]
@sanjusam00427 Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by gautamkrishnar** [comment]
@moollaza  i will inform him and let u know

**Comment by sanjusam00427** [comment]
I will do it , was little busy

**Comment by moollaza** [comment]
@sanjusam00427 thanks again for creating the VirtualBox! You’ve been doing a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

File: share/goodie/cheat_sheets/json/virtualbox.json (hunk 5 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/json/virtualbox.json]: @sanjusam00427 once that's done, this PR Will be able to deploy to Beta for final review and then be merged to go live!

**Comment by moollaza** [comment]
@sanjusam00427 Ping! I'm just checking in again to see if you're still interested in finishing this one up?

It looks like all that's required to get this deployed + merged is changing the filename to `virtual-box.json`

**Comment by mintsoft** [comment]
@sanjusam00427 I'm going to close this for now; feel free to ping myself to reopen it if you get a chance to finish it off :+1:

**Comment by gautamkrishnar** [comment]
@moollaza  @mintsoft  i will finish it up...

**Pull request closed without merging**